crates/youki/src/commands/checkpoint.rs

```diff
@@ -0,0 +1,24 @@
+//! Contains functionality of pause container command
+use crate::commands::load_container;
+use std::path::PathBuf;
+
+use anyhow::{Context, Result};
+
+use liboci_cli::Checkpoint;
+
+pub fn checkpoint(args: Checkpoint, root_path: PathBuf) -> Result<()> {
+    log::debug!("start checkpointing container {}", args.container_id);
+    let mut container = load_container(root_path, &args.container_id)?;
+    let opts = libcontainer::container::CheckpointOptions {
```

Comment:
Why don't you implement `impl CheckpointOption from Checkpoint`?

Reply:
I don't understand this question. Maybe my rust knowledge is not good enough. Can you be more specific about what needs to change here?

Reply:
@adrianreber I'm sorry for my inadequate explanation. Why don't you implement this trait? I feel it would be more Rust-like if the conversion from `Checkpoint` to `CheckpointOptions` could be done using `from()`. WDYT?
https://doc.rust-lang.org/std/convert/trait.From.html

Reply:
Ah, okay. I tried it, but I am not sure where. If I try to implement it in the youki create I get:
```
impl doesn't use only types from inside the current crate
```
If I add it to the liboci_cli crate I need to access information from libcontainer and also the other way around. Not sure if that is desired or even possible.

Any recommendations where I need to add the corresponding implementation?